torch/csrc/jit/serialization/export.h

```diff
@@ -49,7 +54,10 @@ export_onnx(
     const std::map<std::string, int>& custom_opsets = {},
     bool add_node_names = true,
     bool use_external_data_format = false,
-    const std::string& onnx_file_path = std::string());
+    const std::string& onnx_file_path = std::string(),
+    // const std::vector<std::string>& module_names_to_function = {},
```

Comment:
Just a mark: Let's remove such unnecessary variable in final version of this PR.